### PR TITLE
moved solutions line in html course

### PR DESCRIPTION
**Pull request by codyloyd**
more of the same

### CODE REVIEW

**Comment by codyloyd** [comment]
@KevinMulhern chekkkit out man

File: html_css/project_backgrounds.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ In this project you'll clone the website of one of the most design-forward compa
 
 *Send us your solution so we can show others! Submit a link to the Github repo with your files in it by using any of the methods listed on the [contributing page](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md).  See the [Google Homepage project](/web-development-101/html-css) for examples*
 
+* Add your solution below this line!*
```
Comment (KevinMulhern) [inline html_css/project_backgrounds.md]: Theres an extra '*' 😄

Comment (codyloyd) [inline html_css/project_backgrounds.md]: curses

Comment (KevinMulhern) [inline html_css/project_backgrounds.md]: happens to us all 🐐

**Comment by KevinMulhern** [comment]
woo 🎉 one more

**Pull request merged**